src/hotspot/share/compiler/disassembler.cpp

```diff
@@ -837,6 +837,8 @@ bool Disassembler::load_library(outputStream* st) {
   if (_library != NULL) {
     _decode_instructions_virtual = CAST_TO_FN_PTR(Disassembler::decode_func_virtual,
                                           os::dll_lookup(_library, decode_instructions_virtual_name));
+  } else {
+    log_warning(os)("Try to load hsdis library failed");
```

Comment:
NIT: I suggest adding the name of the file that was searched for and rewording this message a bit: "Failed to load hsdis library: hsdis-x86.so"

Reply:
@JohnTortugo 
Thank you for your advice !!
I added the name of the file to the warning message.

Reply:
NIT: "Failed to load hsdis library." or "Loading hsdis library failed."

Reply:
Thank you for your advice.
I changed the warning message to "Loading hsdis library failed".